bowlingGame.js

```diff
@@ -0,0 +1,68 @@
+class BowlingGame {
+  constructor(rollList = []) {
+    this.rollList = rollList;
+    this.sum = 0;
+  }
+
+  roll(pins) {
+    return this.rollList.push(pins);
+  }
+
+  isAStrike(i) {
+    return this.rollList[i] == 10;
+  }
+
+  isASpare(i) {
+    return this.rollList[i] + this.rollList[i + 1] == 10;
+  }
+
+  calculateStrikeScore(i) {
+    console.log("calculating strike score");
+    return (this.sum +=
+      this.rollList[i] + this.rollList[i + 1] + this.rollList[i + 2]);
```

Comment:
You are returning and assigning to an instance variable at the same time. 
It's good to think of class methods at able to do two things:

* changing the state of the system (changing an instance variable)
* calculating something based on the state of the system and either returning it or printing it.

In a well organised class, as much as possible, it's good to have methods that only do one of these things.
Here because the name of the method is calculateStrikeScore, it clearly seems to belong in the first category, so I would prefer it to just return that strike score, maybe:
```
return this.rollList[i] + this.rollList[i + 1] + this.rollList[i + 2];
```